pintc/tests/interfaces/simple.pnt

```diff
@@ -52,6 +52,10 @@ predicate Simple {
     state x = FooInstance::storage::x;
     state y = FooInstance::storage::y;
     state z = BarInstance::storage::z;
+    constraint x == x;
+    constraint y == y;
+    constraint z == z;
+
```

Comment:
So I hate to make you do more work, but in the **near** future, we will almost definitely add a new pass that will optimize away trivial constraints like this. Therefore, it's probably a better idea to make these truly _not dead_. For example:
```rust
constraint x == 0;
constraint y == 0;
constraint z == 0;
```
and so on... 
for complex types like tuples and arrays, you can always do something like:
```
constraint t.0.1 == 0;
```
assuming `t.0.1` is an integer. That is you don't really need to constrain the whole complex variable.